frontend/packages/dev-console/src/components/import/pipeline/PipelineSection.tsx

```diff
@@ -59,7 +59,7 @@ const PipelineSection: React.FC<PipelineSectionProps> = ({ flags }) => {
           <Alert
             isInline
             variant="info"
-            title="Select a builder image to see if there is a pipeline template available for this runtime."
+            title="Select a builder image and resource to see if there is a pipeline template available for this runtime."
```

Comment:
Need to handle some preview information to help the user. [Designs](https://docs.google.com/document/d/1mrNqDt8SzTafBf-sKfqykKy-cHV4py7WII0rTNcyA3c/edit?disco=AAAAJOTSk6U). 

We should have the information from the form to help build this. If you don't have a builder image, we can show this generic message.

Probably the easiest way to test this is to select a builder image that doesn't have one of the templates... like `Nginx`

Reply:
Ah, sorry, this is not the right Alert. The one in `PipelineTemplate` is the one that needs the advanced wording.

Reply:
Updated the text to be more specific using builder image and resources value from the form.

Reply:
But this one here doesn't need to be changed as this is the empty state when no builder image is selected. I've updated this text according to UX.

Reply:
Yup sorry, added the comment and then saw it was the wrong one... wasn't a spot for my to make the other comment so I left it -- was lazy, sorry about that.